src/components/preview/components/page-preview-frame-layout.tsx

```diff
@@ -160,11 +160,21 @@ class PagePreviewFrameLayout extends React.Component<Props, State> {
           break;
           case "layout_gravity":
             property.value.split("|").forEach(gravityValue => {
+              const layoutWidth = parseFloat(String(result.width));
```

Comment:
This depends on the order of properties and it is not guaranteed to be always same. Instead you need to resolve width and height from properties, not from result

Reply:
There is an entirely new implementation which uses flex and doesn't require width and height calculations